### PR TITLE
build images and push into ghcr.io registry

### PR DESCRIPTION
**Pull request by okartau**
pushing to ghcr.io, this time with added workflow instead of modifying main CI flow.

### CODE REVIEW

File: .github/workflows/build-push.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+name: build-push
+on:
+  push:
+    branches:
+      - main
+      - 'release-*'
```
Comment (mythi) [inline .github/workflows/build-push.yml:6]: we can skip `release-` if we hard-code `devel` tag?

Comment (okartau) [reply .github/workflows/build-push.yml:6]: true, I will remove. In this version we can push devel tag only. If we want to push release-xx versions, we need to provide mapping from branch name to tag, but lets handle that part in some next step.

Comment (okartau) [reply .github/workflows/build-push.yml:6]: by same logic, we can also skip trigger on pull_request in this variant.

Comment (okartau) [reply .github/workflows/build-push.yml:6]: but I still want to keep the "if" checks for these in code, as we will need those in final version (embedding this into CI flow). I hope the "if" part still works even when trigger source is removed...

Comment (okartau) [reply .github/workflows/build-push.yml]: New version pushed, removed pull_request trigger, that likely will mean it is not tested here at all. But push anyway cant be tested in PR so thats ok.

**Review by mythi — APPROVED** [review]

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**